Backend/app.py

```diff
@@ -52,67 +54,99 @@ def getDatabaseClient():
   on if we are in Dev or Prod
   '''
   if prod == 'True':
+    logger.info("Using AWS Prod Database")
     return boto3.resource('dynamodb')
   else:
-    print("Returning the Dev resource")
+    logger.info("Using Local Dev Database")
     return boto3.resource('dynamodb', endpoint_url='http://localhost:8000')
 
 @auth.verify_password
 def verify_password(email_or_token, password):
+  logger.info(str(request))
   dynamoDBInstance = getDatabaseClient()
-  return MentiiAuth.verify_password(email_or_token, password, dynamoDBInstance, appSecret)
+  response = MentiiAuth.verify_password(email_or_token, password, dynamoDBInstance, appSecret)
+  logger.info(str(response))
```

Comment:
More detail could be useful later in terms of what is logged instead of just True or False